central/declarativeconfig/updater/group_updater.go

```diff
@@ -0,0 +1,33 @@
+package updater
+
+import (
+	"context"
+
+	"github.com/gogo/protobuf/proto"
+	groupDataStore "github.com/stackrox/rox/central/group/datastore"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/errox"
+)
+
+// GroupUpdater handles updates of storage.Group within declarative config reconciliation routine.
+type GroupUpdater struct {
```

Comment:
no reason to export those, right?
```suggestion
type groupUpdater struct {
```